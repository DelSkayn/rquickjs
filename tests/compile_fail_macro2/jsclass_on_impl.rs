use rquickjs::*;

struct Foo;

#[jsclass]
impl Foo {}

fn main() {}
