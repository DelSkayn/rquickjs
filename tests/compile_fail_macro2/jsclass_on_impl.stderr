error: #[jsclass] only supports structs and enums.
 --> tests/compile_fail_macro2/jsclass_on_impl.rs:6:1
  |
6 | impl Foo {}
  | ^^^^^^^^^^^
